### PR TITLE
Feature/library page

### LINKED ISSUE #2: Library page
## Value Proposition
As a tutee[^1], I would like to have an overview of the tutorials available to me so I can choose which content I want to learn.

## Description
capstone-userstory-01

## Acceptance Criteria
- [x] shows the headline 'MY TUTORIALS'
- [x] shows a list with names and images of all available tutorial
- [x] shows the message "No more tutorials"

## Tasks
- [x] create a new branch
- [x] create a `LibraryPage` component
- [x] create a `TutorialList` and a `TutorialListItem` component
- [x] add `h2` 'MY TUTORIALS'
- [x] create an array with demo content:
>```
>const initialTutorials = [
>  {
>    'id': 'b74d7e1c-acc4-4f65-bfd9-aeaa70dd6878',
>     'name': 'baking bread',
>     'img': 'url'
>  },
>  {
>    'id': '6c3158b7-908a-4472-8379-34296a31009d',
>     'name': 'building bunk bed',
>     'img': 'url'
>  },
>  {
>    'id': '01a29dba-6feb-401c-85eb-77f58d5761d8',
>     'name': 'bathing hamster',
>     'img': 'url'
>  },
>]
>```
- [x] map over this array to create a `TutorialListItem` for each

[^1]: a person who is tutored

### PR DESCRIPTION
**Pull request by kai-franke**
Please check my newly created `TutorialList` component and `library` page. I also added some demo content to a `db.js` and some basic styling to the components.

### CODE REVIEW

**Comment by kai-franke** [comment]
[Preview](https://capstone-project-nou56x3pe-kai-franke.vercel.app/library)

File: components/TutorialList.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+export default function TutorialList({ tutorials }) {
+  return (
+    <ListContainer>
+      {tutorials.map((tutorial) => {
+        return (
```
Comment (F-Kirchhoff) [inline components/TutorialList.js]: You might use an implicit return here

File: components/TutorialList.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const ListContainer = styled.ul`
+  padding: 1em;
+  list-style: none;
+  background-color: var(--background);
+`;
+
+const ListItem = styled.li`
+  background-color: var(--primary-50);
+  display: flex;
+  flex-wrap: nowrap;
```
Comment (F-Kirchhoff) [inline components/TutorialList.js]: this is the default so you can remove it

File: components/TutorialList.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  margin: 1em 0;
```
Comment (F-Kirchhoff) [inline components/TutorialList.js]: maybe you want to space the elements with a grid+gap on the ListContainer

File: components/TutorialList.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  box-shadow: 0px 12px 16px -4px rgba(16, 24, 40, 0.08),
+    0px 4px 6px -2px rgba(16, 24, 40, 0.03);
+`;
+
+const ListImage = styled.figure`
+  position: relative;
+  aspect-ratio: 1;
+  width: 5.625em;
+  height: 5.625em;
```
Comment (F-Kirchhoff) [inline components/TutorialList.js]: This is redundant, you can omit the height

File: db.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      cover: "/assets/tutorial-2_cover-image_2000x2000.jpg",
+    },
+    {
+      id: "01a29dba-6feb-401c-85eb-77f58d5761d8",
+      name: "Building a helpful robot",
+      cover: "/assets/tutorial-3_cover-image_2000x2000.jpg",
+    },
+  ];
+
+  export default initialTutorials
```
Comment (F-Kirchhoff) [inline db.js:19]: nice that you used a db.js

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Great! Only found some minor CSS improvements

File: components/TutorialList.js (hunk 2 above)
Comment (kai-franke) [reply components/TutorialList.js:36]: :white_check_mark:

File: components/TutorialList.js (hunk 1 above)
Comment (kai-franke) [reply components/TutorialList.js]: :white_check_mark:

File: components/TutorialList.js (hunk 3 above)
Comment (kai-franke) [reply components/TutorialList.js]: ✅

File: components/TutorialList.js (hunk 4 above)
Comment (kai-franke) [reply components/TutorialList.js]: ✅

File: components/TutorialList.js (hunk 2 above)
Comment (kai-franke) [reply components/TutorialList.js]: Changed to grid. ✅

File: db.js (hunk 5 above)
Comment (kai-franke) [reply db.js:19]: I had already seen your hint about this in other PR, so I changed it in my case before. ;)

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

**Pull request merged**